### DISCUSSION TITLE
how to display text in html format in v-textarea element?

### DISCUSSION THREAD

**Discussion by timoshechkin** [vuetifyjs/vuetify#16412] [Q&A]
Hello!
how to display text in html format in v-textarea element?

**Comment by hslee2008** [comment]
I think you are trying to type HTML code into the textarea.
There might be two ways to achieve this.

1. You make your own textarea editor component (or use one from npm)
2. Or you could parse markdown (which may include HTML tags) and display as a div

```vue
<template>
  <v-textarea v-model="html"></v-textarea>

  <div v-html="html"></div>
</template>

<script>
const html = ref("<h1>Hello World</h1>");
</script>
```
